libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -1046,6 +1077,12 @@ unsigned CompilerUtils::loadFromMemoryHelper(Type const& _type, bool _fromCallda
 	return numBytes;
 }
 
+shared_ptr<IntegerType> CompilerUtils::fixedPointTypeAsIntegerType(FixedPointType const &_type)
+{
+//	return IntegerType(_type.numBits(), _type.isSigned() ? IntegerType::Modifier::Signed : IntegerType::Modifier::Unsigned);
+	return make_shared<IntegerType>(_type.numBits(), _type.isSigned() ? IntegerType::Modifier::Signed : IntegerType::Modifier::Unsigned);
```

Comment:
Yep, this is what I would have suggested :)
By the way, this function could be handy as part of the `FixedPointType` class.